src/gatsby-plugin-theme-ui/index.ts

```diff
@@ -16,7 +16,7 @@ const breakpoints = [
 const fonts = {
   serif: "'Merriweather', Georgia, Serif",
   sansSerif:
-    "'SF Pro Display', '-apple-system', 'BlinkMacSystemFont', 'San Francisco', 'Helvetica Neue', 'Helvetica', 'Ubuntu', 'Roboto', 'Noto', 'Segoe UI', 'Arial', sans-serif",
+    "'Sen', 'SF Pro Display', '-apple-system', 'BlinkMacSystemFont', 'San Francisco', 'Helvetica Neue', 'Helvetica', 'Ubuntu', 'Roboto', 'Noto', 'Segoe UI', 'Arial', sans-serif",
```

Comment:
Sen?

Reply:
It's a new font, I will adjust the order so on mac we show SF first

https://fonts.google.com/specimen/Sen